<launch>

  <arg name="name" default="artefact_gt" />

  <group>

    <node pkg="image_labeling_utils" type="artefact_gt_publisher.py" name="artefact_gt" output="screen" >
      <rosparam param="objects_keywords">[rescue_randy, phone, backpack, rope, helmet]</rosparam>

      <param name="sdf_path" value="$(find image_labeling_utils)/mrs_simple_cave_01.sdf"/>
      <param name="objects_frame" value="world"/>
    </node>

    <node pkg="tf2_ros" type="static_transform_publisher" name="world2subtpub" output="screen" args="10 0 0 0 0 0 world subt" />
  </group>

</launch>
